### PR TITLE
Issue #13345: Enable examples tests for JavadocTagContinuationIndenta…

### PR DESCRIPTION
**Pull request by AmitKumarDeoghoria**
### Title: Enable examples tests for JavadocTagContinuationIndentationCheck
**Related Issue: #13345** 

**Summary**
This pull request aims to enable example tests for the **JavadocTagContinuationIndentationCheck** in the project. The implementation focuses on ensuring that JavadocTagContinuationIndentation checks are correctly validated through comprehensive test cases, improving overall code reliability and quality.

**Key Changes Made**

- Conversion of txt file to Java File
- Enable the test cases by adding expected violations in the expected array for all example tests
- Made corresponding changes in the xml and xml.template file

### CODE REVIEW

**Comment by AmitKumarDeoghoria** [comment]
GitHub, generate website

**Review by romani — CHANGES_REQUESTED** [review]
items

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadoctagcontinuationindentation/Example2.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadoctagcontinuationindentation;
+
+// xdoc section -- start
+/**
+ * @tag comment
+ * Indentation spacing is 0
+ *   Indentation spacing is 2
+ *  Indentation spacing is 1
```
[the remark this answers is not in the document]
Comment (AmitKumarDeoghoria) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadoctagcontinuationindentation/Example2.java]: Done

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadoctagcontinuationindentation/Example3.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+ * <p> 'p' tag is unclosed
+ * <p> 'p' tag is closed</p>
+ */
+class Example3 {
+  /**
+   * @tag comment
+   *     Indentation spacing is 4
+   */
+  public void testMethod1(String input) {
+    // violation 9 lines above 'Unclosed HTML tag found: p'
```
Comment (romani) [inline src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadoctagcontinuationindentation/Example3.java:23]: Please move this violation above javadoc comment with this problem. I found it it is ok to have such extra lines at the beginning or end, it still easy to catch diff

**Review by romani — CHANGES_REQUESTED** [review]
Absolutely last.

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadoctagcontinuationindentation/Example3.java (hunk 2 above)
Comment (AmitKumarDeoghoria) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadoctagcontinuationindentation/Example3.java]: Done

**Comment by AmitKumarDeoghoria** [comment]
GitHub, generate website

**Comment by romani** [comment]
CI failure https://app.circleci.com/pipelines/github/checkstyle/checkstyle/28861/workflows/c355e960-c032-4b94-9b6e-ff9182952442/jobs/729636/artifacts please try to fix

**Comment by AmitKumarDeoghoria** [comment]
It should be fine now.

**Review by romani — APPROVED** [review]
Thanks a lot.

**Pull request merged**